azext_iot/digitaltwins/params.py

```diff
@@ -287,9 +287,9 @@ def load_digitaltwins_arguments(self, _):
             help="The path to the DTDL component.",
         )
         context.argument(
-            "replace",
-            options_list=["--replace"],
-            help="Indicates the operation should replace an existing twin if it exists."
+            "if_none_match",
+            options_list=["--if-none-match"],
+            help="Indicates the create operation should fail an existing twin if it exists."
```

Comment:
We can make this a bit more clear: "Indicates the create operation should fail if an existing twin with the same id exists."